docs/hugo/content/contributing/add-a-new-code-generated-resource/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Adding a new code generated resource to ASO v2
+linktitle: Add a resource
+layout: single
+---
+
```

Comment:
We should describe the overall process and outputs somewhere e.g. what does the code generator actually generate? Not sure whether that's best here or in one of the first to guides below...